src/main/java/org/generationcp/breeding/manager/listmanager/listcomponent/ListViewActionMenu.java

```diff
@@ -192,11 +186,6 @@ protected void layoutAdminLink() {
 		this.codingAndGroupingOptions.addItem(this.messageSource.getMessage(Message.UNGROUP));
 	}
 
-	@PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_CROP_MANAGEMENT', 'ROLE_LISTS', 'ROLE_GERMPLASM_LISTS', 'ROLE_MG_MANAGE_INVENTORY', 'ROLE_MG_CREATE_LOTS')")
-	private void addCreateInventoryLotsLink() {
-		this.createInventoryLots = this.addItem(this.messageSource.getMessage(Message.CREATE_INVENTORY_LOTS_MENU_ITEM));
```

Comment:
@lik-83 can the Message CREATE_INVENTORY_LOTS_MENU_ITEM  be removed?

Reply:
Done!